docs/DeveloperGuide.md

```diff
@@ -2,8 +2,41 @@
 layout: page
 title: Developer Guide
 ---
-* Table of Contents
-  {:toc}
+### Table of Contents
+- [**Acknowledgements**](#--acknowledgements--)
+- [**Setting up, getting started**](#--setting-up--getting-started--)
+- [**Design**](#--design--)
+  * [Architecture](#architecture)
+  * [UI component](#ui-component)
+  * [Logic component](#logic-component)
+  * [Model component](#model-component)
+  * [Storage component](#storage-component)
+  * [Common classes](#common-classes)
+- [**Implementation**](#--implementation--)
+  * [Add feature](#add-feature)
+  * [Edit feature](#edit-feature)
+  * [Show feature](#show-feature)
+  * [Find feature](#find-feature)
+  * [Filter interview feature](#filter-interview-feature)
+    + [Design considerations:](#design-considerations-)
+  * [Unmark feature](#unmark-feature)
+  * [Datetime for interview](#datetime-for-interview)
+- [**Documentation, logging, testing, configuration, dev-ops**](#--documentation--logging--testing--configuration--dev-ops--)
+- [**Appendix: Requirements**](#--appendix--requirements--)
+  * [Product scope](#product-scope)
+  * [User stories](#user-stories)
+  * [Use cases](#use-cases)
+  * [Non-Functional Requirements](#non-functional-requirements)
+  * [Glossary](#glossary)
+- [**Appendix: Instructions for manual testing**](#--appendix--instructions-for-manual-testing--)
+  * [Launch and shutdown](#launch-and-shutdown)
+  * [Deleting a person](#deleting-a-person)
+  * [Saving data](#saving-data)
+
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>
```

Comment:
Since you are going to edit the table of contents manually anyways, I believe you can just remove this attribution link